### DISCUSSION TITLE
Подключение к серверам игры Code Closers.

### DISCUSSION THREAD

**Discussion by YamatoKaiNi** [bol-van/zapret#756] [Q&A] [answered]
6 ноября 2024 года перестало подключать к игре Code Closers (сайт игры codeclosers.to). С мобильного интернета и через впн подключение проходит, но пинг не лучший. Были определены несколько адресов серверов, которые были прописаны в myblacklist.txt. Данное действие не возымело никакого эффекта. Прошу помощи в пояснении, что я сделал не так

**Comment by HentayHentay** [comment]
Ответ: Стратегия, что у тебя в запрете стоит не совсем подходит для этих сайтов
Что нужно: Проверить эти сайты в блокчеке и найти новую стратегию
Зачем: Если стратегия будет отличаться, что бы можно было применить на конкретных доменах конкретное дурение. 

А ещё там общение может идти на нестандартных портах и протоколах. Тогда только шарка и поиск обращения.

**Reply by YamatoKaiNi** [reply]
У меня навыки в программировании как у хлебушка. Можно языком, который поймет обычный игрок? ПО поводу стратегий. Настроек, как таковых, я точно сделать не смогу

**Reply by HentayHentay** [reply]
Ок
Адреса в каком формате у тебя ?
цифровые или буквенный

**Reply by YamatoKaiNi** [reply]
есть как IP, так и буквенные

**Reply by HentayHentay** [reply]
>  буквенные

Вот это выписываешь или как-то копируешь в блокчек для проверки своей стратегии
1 - Запускаешь данную утилиту и вводишь/вставляешь эти буквенные адреса
2 - дальше жмешь 4 (если имеем 4 блока по 3 цифры)
3 - дальше соглашаешься на проверку всего, что есть
4 - при запросе выбора тестирование - выбираем force
5 - Ожидаем
6 - Стратегии появились 
7 - записываем или копируем. 
8 - сравниваем с теми, что имеем. И если они разные, то делаем лист(текстовый файл)
9 - В этом листе указываем буквенные адреса 
10 - В запрете прописываем стратегию из блокчека и ссылаемся на лист со списком

**Reply by YamatoKaiNi** [reply]
Простите, но какую именно утилиту? У меня, на данный момент, используется Launcher for zapret. И не очень пойму, куда что вводить

**Reply by HentayHentay** [reply]
А там её нет
Присутсвует в отдельном репозитории под [windows](https://github.com/bol-van/zapret-win-bundle)

PS В лаунчере нельзя поменять стратегии. Так что все что выше , то можно забыть сразу. Стратегия там вшита
PS2 Однокнопочные решения они такие. В них нельзя нечего изменить..

**Reply by YamatoKaiNi** [reply]
У меня есть 2 старых версии. Там нет GUI интерфейса. В одном очень много батников (сервис стоп, таск старт и так далее), во втором только сервис (старт, стоп, ремув, файл upd)
Какой именно файл мне нужен?
Я так понял, нужна GoodCheck?

**Reply by HentayHentay** [reply]
любой чекер стратегии на запрет

Старые версии могут не иметь нужного функционала 
Да и сборки могут принести чудеса

**Reply by YamatoKaiNi** [reply]
Вроде понял. Буду пробовать. Инструкция есть. Версию взял GoodCheck 1.3.02 by Ori. Пока ветку не закрываю, могут возникнуть еще вопросы, по типу "правильно ли я брал адреса и искал, куда они ведут.

**Reply by YamatoKaiNi** [reply]
за 20 минут теста (1 прогон поставил) более чем 3/5 не нашло. Значит ли это, что шансы близки к 0 и стоит подумать над приобретением впн или страданием на мобильном инете?

**Reply by HentayHentay** [reply, marked as answer]
Этот вопрос больше к Вам..
